lib/Recommend.js

```diff
@@ -17,6 +17,7 @@ const onboardDomain = 'https://reddit.com';
 
 class Recommender {
   constructor() {
+    this.TelemetryLog = new TelemetryLog();
```

Comment:
Variables should generally be camelCase. TitleCase is generally reserved for class names.